CRM/Upgrade/Incremental/php/FiveSixtySix.php

```diff
@@ -83,6 +84,17 @@ public function upgrade_5_66_beta1($rev): void {
     }
   }
 
+  /**
+   * Upgrade step; adds tasks including 'runSql'.
+   *
+   * @param string $rev
+   *   The version number matching this function name
+   */
+  public function upgrade_5_66_1($rev): void {
+    $this->addTask(ts('Upgrade DB to %1: SQL', [1 => $rev]), 'runSql', $rev);
```

Comment:
Test failure relates.

FWIW, if you want a a quicky-generic way to run upgrade-test on a local build, there's this command:

```bash
civibuild ut dmaster 5.45*
```

Reply:
Thanks... I just remembered `set-version` doesn't generate that sql file for stable releases.